modules/transactions/src/main/java/org/apache/ignite/internal/tx/UpdateCommandResult.java

```diff
@@ -36,6 +36,7 @@ public class UpdateCommandResult implements Serializable {
     /** {@code true} if primary replica belongs to the raft group topology: peers and learners, (@code false) otherwise. */
     private final boolean primaryInPeersAndLearners;
 
+    /** The safe timestamp. */
```

Comment:
According to the code, 0 might be assigned to this field instead of the real safe time requested by the command. If this behavior will be retained, please mention here that 0 can be stored in the field and what it means.

Reply:
0 (HybridTimestamp.NULL_HYBRID_TIMESTAMP) means value is 'null'. Not sure it needs addition comment.

Reply:
It could be useful for someone who is new to the code. Up to you